hw/ip_templates/rv_plic/rtl/rv_plic.sv.tpl

```diff
@@ -309,5 +309,8 @@ module ${module_instance_name} import ${module_instance_name}_reg_pkg::*; #(
                [*`_SEC_CM_ALERT_MAX_CYC]))
 
   // Alert assertions for reg_we onehot check
-  `ASSERT_PRIM_REG_WE_ONEHOT_ERROR_TRIGGER_ALERT(RegWeOnehotCheck_A, u_reg, alert_tx_o[0])
+  `ASSERT_PRIM_REG_WE_ONEHOT_ERROR_TRIGGER_ALERT_IN
```

Comment:
This took me an annoyingly long time to debug, but it's causing Verible to fail in slow lint. Fortunately, the names are just short enough to fit on the line :-D
```systemverilog
  // Alert assertions for reg_we onehot check
  `ASSERT_PRIM_REG_WE_ONEHOT_ERROR_TRIGGER_ALERT_IN(RegWeOnehotCheck_A,
                                                    u_reg,
                                                    gen_alert_tx[0].u_prim_alert_sender.alert_req_i)
```

Reply:
I've gone made, sorry :(